scripts/get-files-in-tarballs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Maybe stupid question, but where/why are we using this? 😅

Reply:
See the PR description. I was using this to verify that the files in the tarballs were the same before and after my changes. I want to keep this file around because I have a feeling that we are gonna make adjustments to the prepack scripts soon.

Reply:
yeah, that makes sense, but do we need to check this in here? I'd remove this if we don't need this (regularly)?

Reply:
I can remove it! This PR will persist.